CWE/history/countries/FLK - Falkland Islands.txt

```diff
@@ -1,6 +1,6 @@
 capital = 2131
-primary_culture = british
-culture = english
+primary_culture = english
```

Comment:
Overseas territories should use the Union culture of its overlord so it should be British here.